CONTRIBUTING.md

```diff
@@ -1,39 +1,164 @@
+## Common issues and resolutions
+
+- Using `grape-active_model_serializers`, or any non-Rails server. See
+    [issue](https://github.com/rails-api/active_model_serializers/issues/1258).
+-
 ## How can I help?
 
-Everyone is encouraged to open issues that are affecting you: bugs, ideas, performance problems – everything helps!
+### Filing an issue
+
+Everyone is encouraged to open issues that are affecting you:
+bugs, ideas, documentation, performance problems – everything helps!
+
+#### Before
+
+1. The first place to start is by looking at our [GitHub Issues](https://github.com/rails-api/active_model_serializers/issues).
+
+  - Check if your issue has already reported.
+  - If you find an existing issue report, feel free to add further information to that report.
+
+
+#### Writing
+
+If possible, please include the following information when [reporting an
+issue](https://github.com/rails-api/active_model_serializers/issues/new):
+
+- ActiveModelSerializers version.
+- What are you using ActiveModel Serializers with? Rails? Grape? Other? Which versions?
+- If you are not running the latest version (please check), and you cannot update it,
+  please specify in your report why you can't update to the latest version.
+- Operating system type + version
+- Ruby version with patch level.  And if you're using rvm, rbenv, etc.
+  - Include your ruby -e "puts RUBY_DESCRIPTION".
+- Clearly-written steps to reproduce the issue (i.e. "Show me how to show myself." ), including:
+  - Command line parameters used, if any.
+  - RubyGems code in your Gemfile, if any. Gemfile.lock, if possible
+  - Any configuration you've made
+  - Nature of reported defect (e.g. user name missing, not "It doesn't work."). Is it intermittent?
+  - Any error messages (including stacktrace, i.e. ""Show me the error.")
+  - Things you've tried.
+  - A pull request for your fix would be great.  Code should have tests.
+  - Link to source code, if available
```

Comment:
For consistency, drop the dots at the end of each item of the enumeration (or put a dot at the end of each one).